src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/PopulateBurdenEstimateRepositoryTests.kt

```diff
@@ -0,0 +1,112 @@
+package org.vaccineimpact.api.databaseTests.tests.burdenEstimateRepository
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.vaccineimpact.api.app.repositories.burdenestimates.CentralBurdenEstimateWriter
+import org.vaccineimpact.api.app.repositories.burdenestimates.StochasticBurdenEstimateWriter
+import org.vaccineimpact.api.db.Tables
+import org.vaccineimpact.api.models.BurdenEstimateSet
+import org.vaccineimpact.api.models.BurdenEstimateSetStatus
+import org.vaccineimpact.api.models.BurdenEstimateSetType
+import org.vaccineimpact.api.models.BurdenEstimateSetTypeCode
+import java.time.Instant
+
+class PopulateBurdenEstimateRepositoryTests : BurdenEstimateRepositoryTests()
+{
+
+    @Test
+    fun `can update set status`()
```

Comment:
These previously private functions were all being implicitly tested before, but now they have been exposed as public methods on the class we can test them individually.